frontend/src/pages/Groups/GroupPage/GroupPage.tsx

```diff
@@ -94,8 +94,7 @@ function GroupHub({ group }: GroupHubProps) {
 
     if (resp["data"]["leaveGroup"]["success"]) {
       push("/groups");
-    }
-    else {
+    } else {
```

Comment:
Was this and other formatting stuff suggested by Prettier or something?

Reply:
Yes